### ISSUE TITLE
[TB] Molecules - Image - Images missing alt text

### ISSUE THREAD

**Issue by kburk1997** [finos/a11y-theme-builder#954]
## Problem/Concern
Images are missing alt text, causing Accessibility Insights for Web errors
### Issue
`Ensures <img> elements have alternate text or a role of none or presentation` ([image-alt](https://accessibilityinsights.io/info-examples/web/image-alt))
### Target application
[A11y ThemeBuilder](http://localhost:3000/designSystem/My%20Design)
### Element path
.image-sq > img
### Snippet
`<img src="/sample.jpg">`
### How to fix
    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"
### Environment
Microsoft Edge version 126.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.43.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
### Screenshot
![Screenshot of three images without alt text on Images page highlighted](https://github.com/finos/a11y-theme-builder/assets/11298269/3b40e529-0e83-4689-9b64-212b75f43f99)

## Proposed Solution
If images are meaningful, add alt text. If not, add empty alt text.

**Comment by toni-sharpe** [comment]
For the sake of general accessibility knowledge sharing, leaving `alt` empty [implicitly adds the aria role "presentation"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#technical_summary). [role="presentation"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) is worth knowing about anyway, it has uses other than images.
